bench_method_test.go

```diff
@@ -54,24 +55,18 @@ func benchmarkMethodForTest(t *testing.T, methodString string, p transport.Proto
 }
 
 func TestBenchmarkMethodWarmTransport(t *testing.T) {
-	s := newServer(t)
+	s := newServer(t, nil)
 	defer s.shutdown()
 	s.register(fooMethod, methods.echo())
 
 	tests := []struct {
-		hostPort       string
-		callerOverride string
-		method         string
-		wantErr        string
+		hostPort string
+		method   string
+		wantErr  string
 	}{
 		{
 			hostPort: s.hostPort(),
 		},
-		{
```

Comment:
this looks unrelated to headers -- should this have been handled in a previous diff?